src/radical/pilot/configs/resource_local.json

```diff
@@ -116,7 +116,7 @@
         "lfs_path_per_node"           : "/tmp",
         "lfs_size_per_node"           : 1024,
         "cores_per_node"              : 8,
-        "gpus_per_node"               : 1
+        "gpus_per_node"               : 2
```

Comment:
Is this going to create a problem if `localhost` has indeed only 1 GPU as most of the "usual" localhost have?

Reply:
It should not - but let me convert it.

BTW, might be good to know: if `RADICAL_DEBUG` is set, you can request any number of cpus, cores etc. from localhost - the FORK RM will consider the machine to be anything you like.  But since this is not the default setting (and should not be), let me fix this.  Thanks.

Reply:
BTW: we configure localhost also to have 8 cores, which is not common - but let's leave it until it becomes a problem, ok?